src/Tools/ExternalAccess/OmniSharp.CSharp/Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.CSharp.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;$(NetVS)</TargetFrameworks>
```

Comment:
I have no idea why we'd multi-target this...

Reply:
I'm happy to remove this down to `netstandard2.0` only if you like.

Reply:
cc @JoeRobich do we need to multi-target this? Or is netstandard2.0 fine?

Reply:
I am fine with the O# EA being netstandard2.0.